katas/content/qec_shor/index.md

```diff
@@ -130,21 +130,21 @@ It takes two parameters: the array of `Pauli` constants (`PauliI`, `PauliX`, `Pa
     "title": "Bit Flip Code"
 })
 
-Can we reuse the ideas of a classical repetition code for a quantum error correction code? 
```

Comment:
Same concern here and for the next two edits: "we" emphasizes that these ideas are the community-wide, not reader-specific